afs-spring-server/src/main/java/com/powsybl/afs/server/io/GzipFilter.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.server.io;
+
+import org.springframework.http.HttpHeaders;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * An http servlet filter which tries to handle automatic gzip compression and decompression:
+ * <ul>
+ *     <li>for POST request with gzip content-encoding, wrap the request to gunzip its content</li>
+ *     <li>Wraps responses : responses with gzip content-encoding will get their output stream wrapped in a gzip output stream</li>
+ * </ul>
+ *
+ * The wrapping of response does not work very well:
+ * the gzip output stream needs to be closed, but apparently spring never closes it.
+ * Therefore, this filter closes the stream after the filter chain, but it will break
+ * the streaming of responses ({@link org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody}.
+ * This filter must not be activated on such endpoints.
+ *
+ */
+public class GzipFilter implements Filter {
```

Comment:
missing author

Reply:
The classes with missing authors did not have an "author" mention in the original repo.
Nonetheless, I just added myself as "author" where I have added some contribution compared to the original repo, but I left the files I did not changed without the author mention.